### DISCUSSION TITLE
Adding Dubbed versions of the anime

### DISCUSSION THREAD

**Discussion by LeftfaceKing** [pystardust/ani-cli#1220] [Ideas] [answered]
can you connect aniwave or aniwatch to the server?

**Comment by mrfluffy-dev** [comment, marked as answer]
`ani-cli --dub`

If you are ever in question about what the script can do just run `ani-cli -h`

**Reply by port19x** [reply]
And if you wanna get more wild wit advanced features, `man ani-cli`
